linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_max_life_existing/oval/shared.xml

```diff
@@ -0,0 +1,35 @@
+<def-group>
+  <definition class="compliance" id="accounts_password_set_max_life_existing" version="3">
+    <metadata>
+      <title>Set Password Expiration Parameters</title>
+      <affected family="unix">
+        <platform>multi_platform_all</platform>
+      </affected>
```

Comment:
Please replace this <affected> tag with the following Jinja2 macro, it handles population of the tag automatically.
      {{{- oval_affected(products) }}}